.gemini/styleguide.md

```
@@ -0,0 +1,47 @@
+# Kiln Code Review Guidelines
+
+### Issues to watch for
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)